lib/ssh/src/ssh_acceptor.erl

```diff
@@ -42,24 +41,10 @@
 %% Internal application API
 %%====================================================================
 %% Supposed to be called in a child-spec of the ssh_acceptor_sup
-start_link(SystemSup, Address, Options) ->
-    proc_lib:start_link(?MODULE, acceptor_init, [self(),SystemSup,Address,Options]).
+start_link(SystemSup, LSock, Address, Options) ->
+    proc_lib:start_link(?MODULE, acceptor_init, [self(), SystemSup, LSock, Address, Options]).
```

Comment:
Here we can use proc_lib:spawn_link or proc_lib:sapwn_opt with link option, instead of start_link to avoid having to use proc_lib:init_ack as all processes under the simple_one_for_one are independent of each other and do not have to wait for each others init functions to complete.